### PR TITLE
Added a note about recoverability on the HA page

### PR DESCRIPTION
**Pull request by milesmatthias**
There aren't any notes about recoverability that I could find in the documentation, so I intended to add one to the high availability page. Feedback appreciated.

### CODE REVIEW

File: reference/halyard/high-availability.md
Comment (dorbin) [inline reference/halyard/high-availability.md]: s/will be able to/can/

Comment (milesmatthias) [reply reference/halyard/high-availability.md]: Thanks @dorbin -- will change that. Does that mean there are further configuration steps needed to allow Spinnaker to recover from failure events?

Comment (dorbin) [reply reference/halyard/high-availability.md]: Nah, not that I know of. Just changing four words, future tense, to one word present tense. :)

**Comment by milesmatthias** [comment]
Updated my note about preemptibles to suggest that it is possible, but not recommended.

cc @viglesiasce

File: reference/halyard/high-availability.md
Comment (dorbin) [inline reference/halyard/high-availability.md]: @milesmatthias 
Rather than follow the statement that it's possible to recover with a statement about what's not recommended, could you make the 2nd sentence be a very brief summary of how to recover? And if it's a multi-step process that needs to be documented explicitly, you could elaborate after this paragraph.

And move the not-recommended statement to a separate paragraph below what *is* recommended?

Comment (milesmatthias) [reply reference/halyard/high-availability.md]: Done @dorbin

**Review by dorbin — COMMENTED** [review]
Nice! Thanks, @milesmatthias 

One last thing, a suggested minor rewrite of your last sentence:
"Also consider hosting Spinnaker on isolated infrastructure to reduce the possibility that other applications or teams will affect your Spinnaker instance."

Other than that, LGTM, pending review from @joonlim

**Comment by milesmatthias** [comment]
Updated @dorbin and merged with upstream master.

Looking forward to any thoughts from @joonlim :)

**Review by joonlim — APPROVED** [review]

**Pull request merged**
